documentation/commands/generate.md

```diff
@@ -4,9 +4,13 @@ _Generates docker compose and terraform files_
 
 Usage: `exo generate [command]`
 
+Flags:
+- `-p, --profile string`  AWS profile to use (defaults to "default"). Only available for `exo generate terraform-var-files`
```

Comment:
Thoughts on actually breaking up this file into three? One for each command? I don't think we need to have them all in one file. They all are subcommands but each is distinct